harmony/harmonytask/task_type_handler.go

```diff
@@ -150,7 +260,8 @@ canAcceptAgain:
 				}
 			}
 			ids = tryAgain
-			goto top
+			b, _ := h.considerWork(from, ids)
```

Comment:
Should we consider work again? Given that BidLogic is single threaded, shouldn't we just dump the win and let someone else take it?

Reply:
We's already spent the effort to:
- decide this is the most important task we can work on 
- decide that we have resources to work on it 
- read a list from SQL
so it seems unfortunate to leave one on the table, and further, it's a priority inversion to abandon this work for other, lower-priority work.